src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -68,34 +68,18 @@ ucs_status_t uct_cuda_ipc_iface_get_device_address(uct_iface_t *tl_iface,
 static ucs_status_t uct_cuda_ipc_iface_get_address(uct_iface_h tl_iface,
                                                    uct_iface_addr_t *iface_addr)
 {
-    uct_cuda_ipc_iface_addr_t *addr = (uct_cuda_ipc_iface_addr_t*)iface_addr;
-    ucs_status_t status;
-    char proc[32];
-
-    addr->pid = getpid();
-
-    uct_ep_get_process_proc_dir(proc, sizeof(proc), addr->pid);
-    status = ucs_sys_get_file_time(proc, UCS_SYS_FILE_TIME_CTIME,
-                                   &addr->starttime);
-    if (status != UCS_OK) {
-        ucs_error("failed to get process starttime");
-        return status;
-    }
-
+    *(pid_t*)iface_addr = getpid();
     return UCS_OK;
 }
 
 static int uct_cuda_ipc_iface_is_reachable(const uct_iface_h tl_iface,
                                            const uct_device_addr_t *dev_addr,
                                            const uct_iface_addr_t *iface_addr)
 {
-    uct_cuda_ipc_iface_t *iface           =
-            ucs_derived_of(tl_iface, uct_cuda_ipc_iface_t);
-    const uct_cuda_ipc_iface_addr_t *addr =
-            (const uct_cuda_ipc_iface_addr_t*)iface_addr;
+    uct_cuda_ipc_iface_t  *iface = ucs_derived_of(tl_iface, uct_cuda_ipc_iface_t);
 
     return ((uct_cuda_ipc_iface_node_guid(&iface->super) ==
-            *((const uint64_t *)dev_addr)) && (getpid() != addr->pid));
+             *((const uint64_t *)dev_addr)) && ((getpid() != *(pid_t *)iface_addr)));
```

Comment:
beautify?

Reply:
this was in previous commits, rolled back changes
if see all files changed - there is no change

Reply:
ok